assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+| Content maintainability    | ✅  |     |     |     |     |
+| Content creation processes | ✅  |     |     |     |     |
+
+### Comments
+
+There are two versions of notary that need to be supported with documentation.
+- V1 is housed with [The Update Framework](https://github.com/theupdateframework/notary), and the docs are included as a set of markdown files in the [/docs folder](https://github.com/theupdateframework/notary/tree/master/docs)
+- V2 is housed with The Notary Project (https://github.com/notaryproject/notaryproject) 
+
+**Information Architecture**: 
+- As there is no website as yet, and development is still ongoing, V2's Project Documentation is all contained within markdown files in the notaryproject repo.
+
+**New user content**:
+- What is there does a good job of explaining the project, but doesn't provide many docs focused on new users.
+
+**Content maintainability**:
+- The docs currently aren't versioned or localized, and is depending on GitHub for search.
+
+**Content creation processes**:
+- There are links to community discussions, but as there is no website yet there is no space to point to regarding docs creation.
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The notaryproject.dev site should primarily focus on the V2 work, but V1 needs to be represented as well.
+- V1 content seems quite good, but looks like it was developed for a site that no longer uses it.
+- For MVP, I'd suggest focusing on V2 content, and linking out to the existing V1 documentation in the location that is currently rather than incorporating it right off the bat.
+
+Eventually I'd expect that V1 would be moved to the Notary Project org, but that is a project level consideration. Even if the whole repo doesn't get moved, there may be value in moving the docs out of TUF's notary repo and into the notaryproject/notaryproject.dev repo.
+
+## Contributor documentation 
+
+**Note**, as my recommendation above is to link to the V1 documentation as is for the website MVP, the remainder of this assessment will only be looking at the existing V2 documentation.
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+### Comments
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+
+**Communication methods documented**
```

Comment:
Once again, the [criteria](https://github.com/cncf/techdocs/blob/main/assessments/criteria.md#communication-methods-documented) are helpful here.

While they do have their communication methods documented, compared to the example provided they could stand to do more :)